lib/addRemove.js

```diff
@@ -64,7 +64,7 @@ function addAsync(version, useNow) {
 			let versionDir = nvsUse.getVersionDir(version);
```

Comment:
Their semantic meaning is "indentation level", and just like font choice and syntax highlighting isn't forced on you, indentation width (a semantic thing, not an aesthetic thing) shouldn't be either. Why would you choose spaces over tabs?? :-D